src/core/Face3.js

```diff
@@ -3,20 +3,20 @@ import { Vector3 } from '../math/Vector3.js';
 
 class Face3 {
 
-	constructor( a, b, c, normal, color, materialIndex ) {
+	constructor( a, b, c, normal, color, materialIndex = 0 ) {
 
 		this.a = a;
 		this.b = b;
 		this.c = c;
 
+		this.materialIndex = materialIndex;
+
 		this.normal = ( normal && normal.isVector3 ) ? normal : new Vector3();
 		this.vertexNormals = Array.isArray( normal ) ? normal : [];
 
 		this.color = ( color && color.isColor ) ? color : new Color();
 		this.vertexColors = Array.isArray( color ) ? color : [];
 
-		this.materialIndex = materialIndex !== undefined ? materialIndex : 0;
-
```

Comment:
Do not change the order of the code please.

Reply:
> Do not change the order of the code please.

Thank you your reply. I change it.